### PR TITLE
Add support for automated documentation

### LINKED ISSUE #38: Add support for autogenerated documentation
Support should be added for automatically generating documentation based on Python docstrings.

### PR DESCRIPTION
**Pull request by sondreo**
Closes #38.

### CODE REVIEW

File: docs/source/modules.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Project modules
```
Comment (oysand) [inline docs/source/modules.rst:1]: Consider removing this as the modules are not entered here yet. Should it prove necessary with this section we can add it at a later point.

File: README.md [hunk 2]
```diff
@@ -94,6 +94,17 @@ After following the steps in [Development](#dev), you can run the tests:
 $ pytest -n 10
 ```
 
+## Documentation
+
+To build the project documentation, run the commands:
```
Comment (oysand) [inline README.md:99]: run the following commands?

Comment (sondreo) [reply README.md]: Agree, has been reworded.

File: docs/source/modules.rst (hunk 1 above)
Comment (sondreo) [reply docs/source/modules.rst]: Removed.

**Pull request merged**